libs/blocks/quiz/quiz.css

```diff
@@ -0,0 +1,377 @@
+.quiz-page * {
+  box-sizing: border-box;
+}
+
+.quiz-page .feds-topnav-wrapper {
+  padding: 0 20px;
```

Comment:
Why are you trying to adjust gnav for quiz pages?

Reply:
Hi @rgclayton,

Here's a thread between Rares and Cody in an earlier PR. Without it, our customized nav has the logo and sign-in hugging the edge uncomfortably. The quiz pages normally use a very minimalistic nav. 

https://github.com/adobecom/milo/pull/1219#discussion_r1317520632